.github/workflows/post-integration-and-release.yml

```diff
@@ -1,12 +1,12 @@
-name: Release
+name: Post-Integration and Release
 
 on:
   push:
     branches:
       - main
 
 jobs:
-  deploy:
+  release:
```

Comment:
Why do we push to both test and "prod" pypi in the same flow - wouldnt this be nice to split up in different flows? E.g. post-integration pushes to test-pypi and release pushes to pypi.

We have the stable step, so we can accumulate changes before we make a release (in order not to spam people).

In next version of this setup, we would want the documentation to be updated.

Reply:
Right now, i think it would be best to align the releases processes - until we have time to streamline to new flow/setup.

Reply:
Ok, let's do it your way. post-integration is now equal to publishing to TestPyPi. Release is publishing to PyPi